cmd/cspc-operator/operator.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
we shouldn't check-in the above binary file ^^cspc-operator

Reply:
Oh! I see.
I will remove that.

Reply:
done